data/campaigns/Heir_To_The_Throne/utils/abilities.cfg

```diff
@@ -107,6 +107,13 @@ _"This unit's grasp of melee tactics allows adjacent allies to strike the first
                 {INITIATIVE_OBJECT x,y=$this_item.x,$this_item.y}
             [/do]
         [/foreach]
+        [lua] # Can't see why {INITIATIVE_OBJECT} leaves the last unit selected.
+            code=<<
+            wesnoth.deselect_hex()
+            -- Take THAT, you stupid bug! **SMASH**
+            -- (Gotta love the 10# sledge when you really should use a tap hammer!)
+            >>
+        [/lua]
 
```

Comment:
I notice that line 44 of object.lua selects the unit; I wonder if that's the cause of this bug. Though, I also wonder whether removing it would give the expected behaviour for the [object] popup...